.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,62 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ""
+labels: bug
+title: ""
+---
+
+<!--
+If you are reporting a new issue, make sure that we do not have any duplicates
+already open. You can ensure this by searching the issue list for this
+repository. If there is a duplicate, please close your issue and add a comment
+to the existing issue instead.
+-->
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```

Comment:
```suggestion
**Add the steps to Reproduce**
```

Reply:
@aevesdocker I'll be using the suggestion below as it contains a lowercase `r` for the `Reproduce` word:

```suggestion
**Add the steps to reproduce**
```